Compose/NCSA/docker-compose.yml

```diff
@@ -155,17 +115,17 @@ services:
     stdin_open: true
     tty: true
 
-#  atmonochromator:
-#    image: lsstts/monochromator:v0.1.0_sal4.4.0_salobj3.11.1
-#    container_name: atmonochromator
-#    environment:
-#      - LSST_DDS_DOMAIN=auxtelpath
-#    tty: true
-#    networks:
-#      test-stand-network:
-#        ipv4_address: "141.142.238.141"
+##  atmonochromator:
```

Comment:
Can the following code just be removed since it is now double commented out?

Reply:
done